ui-app/client/src/components/MarkdownNavigator/markdownNavigatorProperties.ts

```diff
@@ -0,0 +1,71 @@
+import { SCHEMA_STRING_COMP_PROP } from '../../constants';
+import {
+	ComponentPropertyDefinition,
+	ComponentPropertyEditor,
+	ComponentPropertyGroup,
+	ComponentStylePropertyDefinition,
+} from '../../types/common';
+import { COMMON_COMPONENT_PROPERTIES, COMPONENT_STYLE_GROUP_PROPERTIES } from '../util/properties';
+
+const propertiesDefinition: Array<ComponentPropertyDefinition> = [
+	{
+		name: 'text',
+		schema: SCHEMA_STRING_COMP_PROP,
+		displayName: 'Markdown Text',
+		description: 'Markdown Text',
+		group: ComponentPropertyGroup.BASIC,
+		editor: ComponentPropertyEditor.TEXT_EDITOR,
+		translatable: true,
+	},
+
+	{
+		name: 'showLinksFor',
+		schema: SCHEMA_STRING_COMP_PROP,
+		displayName: 'Show Links For',
+		description: 'Show Links For',
+		editor: ComponentPropertyEditor.ENUM,
+		group: ComponentPropertyGroup.BASIC,
+		defaultValue: 'h1',
+		enumValues: [
+			{
+				name: 'h1',
+				displayName: 'Show Links for H1',
+				description: 'Show Links for H1',
+			},
+			{
+				name: 'h1Andh2',
+				displayName: 'Show Links for H1 and H2',
+				description: 'Show Links for H1 and H2',
+			},
+		],
+	},
+
+	{
+		...COMMON_COMPONENT_PROPERTIES.designType,
+		enumValues: [...COMMON_COMPONENT_PROPERTIES.designType.enumValues!],
+	},
```

Comment:
Just use the COMMON_COMPONENT_PROPERTIES.designType if you don't have any changes